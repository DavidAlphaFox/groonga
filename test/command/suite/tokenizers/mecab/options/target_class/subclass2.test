#@on-error omit
tokenize \
  'TokenMecab("include_class", true, \
              "target_class", "名詞/固有名詞/人名/姓")' \
  '私の名前は中野太郎です。'
#@on-error default
