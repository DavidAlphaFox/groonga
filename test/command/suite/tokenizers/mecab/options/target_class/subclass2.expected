tokenize   'TokenMecab("include_class", true,               "target_class", "名詞/固有名詞/人名/姓")'   '私の名前は山田太郎です。'
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "山田",
      "position": 0,
      "force_prefix": false,
      "force_prefix_search": false,
      "metadata": {
        "class": "名詞",
        "subclass0": "固有名詞",
        "subclass1": "人名",
        "subclass2": "姓"
      }
    }
  ]
]
